packages/@aws-cdk/aws-apigateway/lib/integration.ts

```diff
@@ -134,6 +134,18 @@ export interface IntegrationProps {
   readonly options?: IntegrationOptions;
 }
 
+/**
+ * Result of binding an Integration to the Method
+ */
+export interface IntegrationConfig {
+  /**
+   * Triggers a new Deployment if the value of this property changes.
+   * Use this property to trigger new deployments for changes to values in this integration.
+   * @default - deployments are not triggered for any change of this integration.
+   */
+  readonly triggerDeployment?: any;
```

Comment:
I think using a verb semantics here is weird because it's not an active thing the producer needs to do. Perhaps `deploymentFingerprint`.

I would also force this to be a `string` and have the producer serialize to ensure people understand this needs to be a serializable value and not any random object with cyclic references to the world.